apps/vscode/src/providers/views/Account/Account.tsx

```
@@ -10,8 +10,8 @@
  * the extension host to the pure AccountView component.
  */
 
-import 'shell/src/themes/rocketride-default.css';
-import 'shell/src/themes/rocketride-vscode.css';
+import 'shell/themes/rocketride-default.css';
```

ESLint

[error] 'shell/themes/rocketride-default.css' import is restricted from being used by a pattern. Only shell package sources are deep-importable in-tree (shell/src/<group>); everything else is relative (in-package) or the barrel (elsewhere).

(no-restricted-imports)